app/locales/zh/translations.js

```diff
@@ -100,53 +100,53 @@ export default {
     "97": "97",
     "98": "98",
     "99": "99",
-    "Key": "Chinese Translation",
+    "Key": "Final HK Chinese",
```

Comment:
There are two separate Chinese translations in the ISP translations folder, which makes me think that overwriting the existing chinese translation might not be what we want.

We may want to revert this file (use the original chinese translation), and create a separate `zh-HK` translation file for the Hong Kong version. Long Ze may be able to advise and make sure that *both* of the chinese translation versions look right.